lib/updateFollowers.js

```diff
@@ -3,7 +3,6 @@ module.exports = { updateFollowerCount };
 function updateFollowerCount(content, count) {
   return content.replace(
     /<!-- follower-counter -->(\d+)<!-- \/follower-counter -->/,
-    (_content, counter) =>
```

Comment:
This was not needed since I am not manipulating the count like the nose boop is.